addon/components/lt-infinity.hbs

```diff
@@ -1 +1,3 @@
-{{yield}}
+<div class="lt-infinity">
```

Comment:
Since we are adding this div here, shouldn't we make the JS tagless?

Reply:
This is one of the situations where tagless breaks the addon.

If it's tagless the enter viewport method isn't triggered when scrolling to the bottom.  The table doesn't load any rows.

The ember id/view class can't be attached so it's not  recognized moving into the viewport.  

Tagless:
![image](https://user-images.githubusercontent.com/18253733/183531087-f40efd5f-9b9b-491d-b1f2-96748ae3855d.png)

Not tagless:
![image](https://user-images.githubusercontent.com/18253733/183530947-bfab3a25-33d1-4dce-88b3-57ede93f9ce6.png)

Not sure there is a work around until it's converted to glimmer...

Reply:
Converting to glimmer I don't think would make a difference? We have to refactor away from this behavior either way. Suppose we can wait and do it later though.

Reply:
agree